README.aspnetcore-build-nightly.md

```diff
@@ -35,8 +42,7 @@ This image contains:
 
 # Related images
 
-1. [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) - the .NET Core image if you don't need the ASP.NET Core specific optimizations.
-2. [microsoft/aspnetcore-nightly](https://hub.docker.com/r/microsoft/aspnetcore-nightly/) - the ASP.NET Core runtime image, for when you don't need to build inside a container.
+* [microsoft/aspnetcore-nightly](https://hub.docker.com/r/microsoft/aspnetcore-nightly/) - the ASP.NET Core runtime image, for when you don't need to build inside a container.
 
 # Example Usage
```

Comment:
What do you think about referencing readers to the [aspnet sample](https://github.com/dotnet/dotnet-docker/blob/master/samples/aspnetapp/README.md).  Our samples seem to cover more scenarios and should be the single place we invest in.  Comment applies to all of the readmes.

Reply:
Sure, that would be fine with me.